src/style.css

```diff
@@ -1 +1,73 @@
-/* Styles go here */
+
+@import url(https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap);
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  margin: 0;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: flex;
+  padding: 10px 0;
```

Comment:
but what if header height increase
how to update the padding?

It is better to center vertically